Makefile.am

```diff
@@ -87,13 +110,18 @@ libeos_updater_util_libeos_updater_util_@EUU_API_VERSION@_la_SOURCES = \
 	libeos-updater-util/avahi-service-file.h \
 	libeos-updater-util/config.c \
 	libeos-updater-util/config.h \
+	libeos-updater-util/eos-updater-types.c \
+	libeos-updater-util/eos-updater-types.h \
```

Comment:
Moving `eos-updater-types.[ch]` to `libeos-updater-util` makes sense, but please rename them to drop the prefix. So they should become `libeos-updater-util/types.[ch]`. See my comments about `eos-updater-enums.[ch]` for the reasoning.